flink-runtime/src/main/scala/org/apache/flink/runtime/taskmanager/TaskManager.scala

```diff
@@ -1894,6 +1895,52 @@ object TaskManager {
       override def getValue: Double =
         ManagementFactory.getOperatingSystemMXBean().getSystemLoadAverage()
     })
+    
+    // Preprocessing steps for registering cpuLoad
+    // fetch the method to get process CPU load
+    val getCPULoadMethod: Method = getMethodToFetchCPULoad()
```

Comment:
Why not making `getCPULoadMethod` an `Option[Method]`? That way one avoids vicious `null` values.